### PR TITLE
Blog - Misaligned Incentives in Dev Tool Businesses

### PR DESCRIPTION
**Pull request by msali123**
Done with all checks

### CODE REVIEW

**Comment by msali123** [comment]
[Preview Link](https://6446810f462a852a7abdb897--ephemeral-narwhal-3ac3b2.netlify.app/blog/misaligned-incentives/)

@balapriyac @adamgordonbell

**Review by balapriyac — CHANGES_REQUESTED** [review]
@adamgordonbell @msali123 I've suggested a small formatting change in the quote. Everything else looks great. Also, the article is scheduled for 2nd June, 2023. Should we change this to sometime earlier?

File: blog/_posts/2023-06-02-misaligned-incentives.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+## Open Source Incentives
+
+Vlad mentioned in a meeting early after my joining Earthly that the line between what's open source and what's commercial is tricky. Because at that line, incentives can get messy. What do you do if you sell a commercial feature and someone wants to add something like it to the open-source product? You want to encourage contribution, but you also need to keep the lights on. For this reason, we drew the line at compute and SaaS features. Open-source users can contribute as much as they wish to Earthly, because the monetization is SaaS-based, not feature based.
+
+![COST]({{site.images}}{{page.slug}}/cost.png)\
+
+There is this platitude about how you are the product if you aren't paying for something. It's not 100% true, but it does get at an important point. If you cost a business money, they will want to recoup that at some point. (Even if it means angering everyone by trying to charge for Dockerhub or Docker Desktop.)
```
Comment (adamgordonbell) [inline blog/_posts/2023-06-02-misaligned-incentives.md:36]:
```suggestion
There is this platitude about how you are the product if you aren't paying for something. It's not 100% true, but it does get at an important point. If you cost a business money, they will want to recoup that at some point. (Even if it means angering everyone by trying to charge for [Dockerhub](https://www.theregister.com/2023/03/17/docker_free_teams_plan/) or [Docker Desktop](https://www.theregister.com/2021/08/31/docker_desktop_no_longer_free/).)
```

**Review by adamgordonbell — APPROVED** [review]

**Comment by adamgordonbell** [comment]
> @adamgordonbell @msali123 I've suggested a small formatting change in the quote. Everything else looks great. Also, the article is scheduled for 2nd June, 2023. Should we change this to sometime earlier?

Yeah, the article looks good, and i like the suggested change. I'm going to move the date up and then merge it.

File: blog/_posts/2023-06-02-misaligned-incentives.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+![COST]({{site.images}}{{page.slug}}/cost.png)\
+
+There is this platitude about how you are the product if you aren't paying for something. It's not 100% true, but it does get at an important point. If you cost a business money, they will want to recoup that at some point. (Even if it means angering everyone by trying to charge for [Dockerhub](https://www.theregister.com/2023/03/17/docker_free_teams_plan/) or [Docker Desktop](https://www.theregister.com/2021/08/31/docker_desktop_no_longer_free/).)
+
+TailScale is the only company I've seen talk explicitly about the incentives around free products and what they get out of free users:
+
+> Perhaps we're not supposed to say the quiet part out loud, but it's important for the discussion. Our architectural decisions were made carefully, and are paying off.
```
Comment (adamgordonbell) [inline blog/_posts/2023-06-02-misaligned-incentives.md:40]:
```suggestion
> Perhaps we're not supposed to say the quiet part out loud, but it's important for the discussion. Our architectural decisions were made carefully, and are paying off.
>
```

**Pull request merged**